lib/services/dialog/index.ts

```diff
@@ -35,38 +35,40 @@ export const utils = {
   isIntentInScope,
 };
 
-declare type DMStore = {
+export declare type DMStore = {
```

Comment:
not sure why this was a `declare type` to begin with, i would remove that
```suggestion
export type DMStore = {
```